Doc/library/graphlib.rst

```diff
@@ -194,15 +196,74 @@
    .. versionadded:: 3.9
 
 
+Graph Functions
+---------------
+
+Some functions are provided to work with directed acyclic graph structures using
+the same representation accepted by :meth:`TopologicalSorter.__init__`: a
```

Comment:
Document creation fail here.

```suggestion
the same representation accepted by :meth:`!TopologicalSorter.__init__`: a
```

I'm not sure if that's right

Reply:
Perhaps just better to say:

```suggestion
the same representation accepted by the :class:`TopologicalSorter` constructor: a
```

or 


```suggestion
the same representation accepted by :class:`TopologicalSorter`: a
```

Reply:
Huh, I don't see an error from `make -C doc html`.

Reply:
We have a slightly odd setup to baseline reference warnings (see `check-warnings.py`), so CI runs once normally and then a second time with `-n` as an option to Sphinx.